conf/modules.config

```diff
@@ -168,6 +180,7 @@ process {
             mode: params.publish_dir_mode,
             saveAs: { filename -> filename.equals('versions.yml') ? null : filename }
         ]
+        ext.args = "--keep-negatives"
```

Comment:
Just to confirm: we definitely want to hardcode this in the module?

Reply:
This is to get all possible amp and non amps and then apply the threshold that the user specifies in ampcombi. There we have a threshold flag that is applied across all the tool results. This way the user doesnt have to change multiple parametrs for multiple modules.